browser/ui/webui/skus_internals_ui.cc

```diff
@@ -94,25 +95,36 @@ void SkusInternalsUI::GetVpnState(GetVpnStateCallback callback) {
 #if !BUILDFLAG(IS_ANDROID)
   dict.Set("Last connection error", GetLastVPNConnectionError());
 #endif
-  dict.Set("Order", GetVPNOrderInfo());
+  auto* profile = Profile::FromWebUI(web_ui());
+  if (!brave_vpn::IsBraveVPNEnabled(profile->GetPrefs())) {
+    base::Value::Dict empty;
+    dict.Set("Order", empty);
+  } else {
+    auto order_info = GetOrderInfo("vpn.");
+    order_info.Set(
+        "env", local_state_->GetString(brave_vpn::prefs::kBraveVPNEnvironment));
+    dict.Set("Order", order_info);
+  }
```

Comment:
Is this change even needed? This part won't evaluate for iOS due to `#if BUILDFLAG(ENABLE_BRAVE_VPN)` check around it

Reply:
Nvm I see it was pulled from `GetVPNOrderInfo`